oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -69,170 +216,153 @@ if __name__ == '__main__':
     # To avoid systemd in container environment
     use_systemd = True
     if len(sys.argv) > 1:
-        use_systemd = not sys.argv[1] == 'no'
-    sm_config = configparser.ConfigParser()
+        use_systemd = sys.argv[1] != 'no'
 
+    sm_config = configparser.ConfigParser()
     sm_config.read('/etc/columnstore/storagemanager.cnf')
+    storage = sm_config.get(
+        'ObjectStorage', 'service', fallback='LocalStorage'
+    )
+    bucket = sm_config.get('S3', 'bucket', fallback='some_bucket')
+    s3_enabled = storage.lower() == 's3' and bucket.lower() != 'some_bucket'
+
     cs_config = ET.parse('/etc/columnstore/Columnstore.xml')
     config_root = cs_config.getroot()
-
-    storage = sm_config.get('ObjectStorage', 'service')
-    if storage is None:
-        storage = 'LocalStorage'
-    bucket = sm_config.get('S3', 'bucket')
-    if bucket is None:
-        bucket = 'some_bucket'
-
-    s3_enabled = storage.lower() == 's3' and not bucket.lower() == 'some_bucket'
-
     dbrmroot = config_root.find('./SystemConfig/DBRMRoot').text
     pmCount = int(config_root.find('./SystemModuleConfig/ModuleCount3').text)
     is_multinode = pmCount > 1
-    loadbrm = '@ENGINE_BINDIR@/load_brm'
-    s3_dbroot1_brm_path = 'data1/systemFiles/dbrm/BRM_saves_current'
-
     brm_saves_current = ''
 
     if s3_enabled:
         # start SM using systemd
         if use_systemd is True:
             cmd = 'systemctl start mcs-storagemanager'
-            retcode = subprocess.call(cmd, shell=True)
-            if retcode != 0:
-                print('Failed to start storagemanager. \
-{} exits with {}.'.format(cmd, retcode))
+            try:
+                subprocess.check_call(cmd, shell=True)
+            except subprocess.CalledProcessError as exc:
+                print(
+                    f'Failed to start storagemanager. {cmd} exits with',
```

Comment:
This format is no go IMHO. I didn't get how didn't it failed but the minimal python version we support doesn't support f'' strings formatting.

Reply:
Can you please clarify, which is has minimal Python without f strings. Centos 7 has python 3.6.8, and f strings are included

Reply:
prove
➜  xbench git:(feature/columnstore-2) ✗ sudo docker run  -it centos:7         
[root@9e6328da92a6 /]# yum install python3                                                                                                                                                    
Loaded plugins: fastestmirror, ovl                                                             
Determining fastest mirrors                                                                                                                                                                   
 * base: mirror.hostnet.nl                                                                                                                                                                    
 * extras: centos.mirror.triple-it.nl                                                                                                                                                         
 * updates: centos.mirror.triple-it.nl                                                                                                                                                        
base                                                                                                                                                                   | 3.6 kB  00:00:00     
extras                                                                                                                                                                 | 2.9 kB  00:00:00     
updates                                                                                                                                                                | 2.9 kB  00:00:00     
(1/4): base/7/x86_64/group_gz                                                                                                                                          | 153 kB  00:00:00     
(2/4): extras/7/x86_64/primary_db                                                                                                                                      | 250 kB  00:00:00     
(3/4): base/7/x86_64/primary_db                                                                                                                                        | 6.1 MB  00:00:00     
(4/4): updates/7/x86_64/primary_db                                                                                                                                     |  17 MB  00:00:00     
Resolving Dependencies                                                                         
--> Running transaction check
---> Package python3.x86_64 0:3.6.8-18.el7 will be installed
--> Processing Dependency: python3-libs(x86-64) = 3.6.8-18.el7 for package: python3-3.6.8-18.el7.x86_64
--> Processing Dependency: python3-setuptools for package: python3-3.6.8-18.el7.x86_64
--> Processing Dependency: python3-pip for package: python3-3.6.8-18.el7.x86_64
--> Processing Dependency: libpython3.6m.so.1.0()(64bit) for package: python3-3.6.8-18.el7.x86_64                                                                                             
--> Running transaction check                                                                                                                                                                 
---> Package python3-libs.x86_64 0:3.6.8-18.el7 will be installed                                                                                                                             
--> Processing Dependency: libtirpc.so.1()(64bit) for package: python3-libs-3.6.8-18.el7.x86_64                                                                                               
---> Package python3-pip.noarch 0:9.0.3-8.el7 will be installed                                                                                                                               
---> Package python3-setuptools.noarch 0:39.2.0-10.el7 will be installed                                                                                                                      
--> Running transaction check                                                                                                                                                                 
---> Package libtirpc.x86_64 0:0.2.4-0.16.el7 will be installed                                                                                                                               
--> Finished Dependency Resolution                                                                                                                                                            
                                                                                                                                                                                              
Dependencies Resolved
                                               
==============================================================================================================================================================================================
 Package                                             Arch                                    Version                                           Repository                                Size
==============================================================================================================================================================================================
Installing:                                                                                                                                                                                   
 python3                                             x86_64                                  3.6.8-18.el7

Reply:
Removed f-strings formatting.